mintapi/signIn.py

```diff
@@ -369,12 +368,14 @@ def sign_in(
         )
         account_selection_page(driver, intuit_account)
         password_page(driver, password)
+        # Give the overview page a chance to actually load
+        time.sleep(5)
```

Comment:
Is it possible to use some kind of [explicit wait](https://www.browserstack.com/guide/selenium-wait-for-page-to-load) here or page load event? A sleep doesn't seem super robust as the load could take longer on a slow connection, or it could be done faster and we're just slowing it down.

Reply:
@mrooney I'll look at what you linked and see. The reason behind this is we don't know for sure if we're going to the main page or looping through to the next round of MFA.